### PR TITLE
fix-mix-es5-with-es6

### PR DESCRIPTION
**Pull request by Lydie260**
### PR Description
This pr fixes the mixture of ES5 and ES6 in the index file
### Description of tasks that were expected to be completed
fixing ES5 with ES6 
### PR Checklist:
- [x] Fix ES5 with ES5.

### CODE REVIEW

**Review by NTElissa — APPROVED** [review]

**Review by Gentille-dev — APPROVED** [review]

**Review by Charlesntwari — APPROVED** [review]

File: src/index.ts [hunk 1]
```diff
@@ -89,33 +90,6 @@ export const typeDefs = mergeTypeDefs([
...
-      },
-    },
-    {
-      // This is the `httpServer` we created in a previous step.
-      server: httpServer,
-      // This `server` is the instance returned from `new ApolloServer`.
-      path: '/',
-    }
-  )
-
```
Comment (ceelogre) [inline src/index.ts]: Why did you remove the subscription API?
